### PR TITLE
Improve Windows development environment

### PR DESCRIPTION
**Pull request by rabstejnek**
Running hawc dev environment on windows can be troublesome.  Iron out a few of the issues identified:

- [x] Build a batch script which acts similarly to the Makefile
- [x] Fix 'manage.py' CLI command for Windows
- [x] Change BMDS dependency to a Windows friendly version
- [x] Add updated install instructions and new batch script instructions

### CODE REVIEW

File: hawc/__init__.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 import sys
 
 
 def manage():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hawc.main.settings.dev")
     from django.core.management import execute_from_command_line
 
+    # Setuptools generate entry point scripts with "-script.py" appended
+    # to the end of them in Windows
+    if platform.system() != "Windows":
```
Comment (shapiromatron) [inline hawc/__init__.py]: shouldn't that be a `==`, not a `!=`?

File: make.bat [hunk 2]
```diff
@@ -0,0 +1,62 @@
+@ECHO off
```
Comment (shapiromatron) [inline make.bat]: this is amazing hackery.

File: hawc/__init__.py (hunk 1 above)
Comment (rabstejnek) [reply hawc/__init__.py]: It should! Thank you for catching that, fixed.

**Comment by rabstejnek** [comment]
@shapiromatron Ready for review

File: docs/source/development.rst [hunk 3]
```diff
@@ -224,3 +224,94 @@ Recommended workspace settings:
...
+    mkdir %HOMEPATH%\dev
+    cd %HOMEPATH%\dev
+    git clone https://github.com/shapiromatron/hawc.git
+    cd hawc
+
+    :: install the python requirements
+    conda activate hawc
+    python -m venv venv
+    venv\Scripts\activate.bat
+    pip install -r requirements\dev.txt
```
Comment (shapiromatron) [inline docs/source/development.rst]: can we update pip before this like? `python -m pip install --upgrade pip`?

**Pull request merged**